common/config.go

```diff
@@ -0,0 +1,273 @@
+/*
```

Comment:
common is imported by both edge_core and edgecontroller. Can we place this file in installer folder as this is only used by the installer for now?

Reply:
Off course we can. But this is done to avoid any future modifications in installer code. Now for any future feature either in cloud or edge, developer can use these library directly which is in common place. Else later if it gets used by cloud or edge module, this code requires to be moved out and then code modifications required in keadm. We don't want these hassles later for developers. So i suggest we keep it as it is.

Reply:
common folder is used if the code is used by cloud and edge both like the device structures. Any code that is useful only to just one should be present in their respective folder. Edge binary need not have code to generate controller config. It is not useful. Binary size of edge is very important. It should be reduced as much as possible.

Reply:
Done